README.md

```diff
@@ -68,6 +68,12 @@ $ curl -H "Accept: text/turtle" \
   http://localhost:3000/myfile.ttl
 ```
 
+Retrieve a turtle file in a different serialization:
+```bash
+$ curl -H "Accept: application/ld+json" \
+  http://localhost:3000/myfile.ttl
```

Comment:
So is this going to keep being a thing to motivate me to add features? ;)

Reply:
I actually forgot I left this there :P
I thought this would already work with this PR.

Reply:
Only if you do a PATCH first with the in-memory store since that changes the content-type internally 😄